File: Core/Entities/MatchedJobCandidate.cs
using System.Collections.Generic;

namespace Core.Entities
{
    public class MatchedJobCandidate
    {
        public int JobId { get; set; }
        public int CandidateId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int WeightingsSum { get; set; }
        public int SkillsCount { get; set; }
        public IList<MatchedSkill> MatchedSkills { get; set; }
    }
}

File: client/src/app/shared/models/matched-job-candidate.ts
export interface IMatchedJobCandidate {
  jobId: number;
  candidateId: number;
  firstName: string;
  lastName: string;
  weightingsSum: number;
  skillsCount: number;
  matchedSkills: IMatchedSkill[];
}

export interface IMatchedSkill {
  jobId: number;
  candidateId: number;
  name: string;
  jobWeighting: number;
  candidateWeighting: number;
}
